torch_geometric/explain/algorithm/utils.py

```diff
@@ -56,3 +56,12 @@ def clear_masks(model: torch.nn.Module):
             module._loop_mask = None
             module._apply_sigmoid = True
     return module
+
+
+def clear_hetero_masks(model: torch.nn.Module):
```

Comment:
`clear_masks` should work for heterogenous models too. so we don't need this function.